chart/values.yaml

```diff
@@ -4587,6 +4587,16 @@ tenantRequirementsChecker:
     requests:
       memory: "128Mi"
       cpu: "250m"
+  ## Configure Pods Security Context
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
+  ## @param tenantRequirementsChecker.podSecurityContext.enabled Enabled Upgrade Check pre-hook pods' Security Context
+  ## @param tenantRequirementsChecker.podSecurityContext.fsGroup Set Upgrade Check pre-hook pod's Security Context fsGroup
+  ## @param tenantRequirementsChecker.podSecurityContext.supplementalGroups[0] Set Upgrade Check pre-hook pod's Security Context supplementalGroups
+  ##
+  podSecurityContext:
+    enabled: true
+    fsGroup: 101
+    supplementalGroups: [2345]
```

Comment:
I think that we're not using this in the pod template no? 🤔 Same happens with the next and previous lines? 😓

Reply:
Yep, you are right, I think the best approach is defining these in the values and then referring them in the preflights, I will do these changes